stories/topics/tech-note-linux-files-owned-by-root.adoc

```diff
@@ -0,0 +1,18 @@
+[id="tech-note-linux-files-owned-by-root"]
+
+= Command generator - Linux files owned by root
+
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created:
+
+[literal, options="nowrap" subs="+attributes"]
+----
+# Change the owner of the command_generator_data directory recursively
+sudo chown -R $USER:$USER command_generator_data/
+
+# Check the permissions
+ls -la command_generator_data/
+----
+
+The command generator currently expects the usage of the Docker CLI in the link:https://docs.docker.com/config/daemon/[daemon mode]. By default, link:https://docs.docker.com/engine/security/userns-remap/#enable-userns-remap-on-the-daemon[Docker do not have User namespace mapping] for Discretionary access control (DAC).
+
+You can learn more about the Linux namespaces, including the User namespace at the article link:https://www.redhat.com/sysadmin/7-linux-namespaces[The 7 most used Linux namespaces].
```

Comment:
For more information about Linux namespaces, including the user namespace, read link:https://www.redhat.com/sysadmin/7-linux-namespaces[The 7 most used Linux namespaces].